.eslintrc.cjs

```diff
@@ -1,7 +1,9 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
```

Comment:
this rule is a bit annoying for certain config object, e.g. here when `root` is usually at the top

Reply:
There is no need for root to be at the top though, is it?

Reply:
No, but it's the common convention from what I've seen. But we didn't need this change so I removed it altogether instead